src/courtroom.cpp

```diff
@@ -4110,6 +4162,20 @@ void Courtroom::on_custom_objection_clicked()
   ui_ic_chat_message->setFocus();
 }
 
+void Courtroom::ShowContextMenu(const QPoint &pos)
```

Comment:
```suggestion
void Courtroom::show_custom_objection_menu(const QPoint &pos)
```

Reply:
this was also caused by porting of 2.7 code verbatim